app/src/main/java/com/jerboa/ui/components/common/TimeAgo.kt

```diff
@@ -144,52 +146,58 @@ fun ScoreAndTime(
             downvotes = instantScores.downvotes,
         )
 
+        // If the show_scores is disabled, and we are the instance is pre 0.19.4, we fallback to legacy behaviour
+        val version = API.getInstanceOrNull()?.version
+        val legacyScoresHidden = version != null && version < Version(0, 19, 4) && !voteDisplayMode.score
```

Comment:
You can probably just use the `!api.FF.hidePost()` like in the other files, to denote that its before `0.19.4`

Reply:
Ah yes